core/src/test/java/kafka/test/annotation/ClusterTest.java

```diff
@@ -44,4 +44,5 @@
     String listener() default "";
     MetadataVersion metadataVersion() default MetadataVersion.IBP_3_8_IV0;
     ClusterConfigProperty[] serverProperties() default {};
+    ClusterConfigDisplayTag[] displayTags() default {};
```

Comment:
We can remove the `name` field now

Reply:
i was wondering if it is being used by anyone now, but just checked and found that this is only used in test. Let me remove that